src/components/Related/DomainsOnProtein/DomainsOnProteinLoaded/index.tsx

```diff
@@ -13,11 +13,19 @@ const ProteinViewer = loadable({
   loading: null,
 });
 
+// 0A017SEX7 is a good example
 const UNDERSCORE = /_/g;
 const FIRST_IN_ORDER = [
   'representative_domains',
-  'secondary_structure',
+  'representative_families', // coming soon
+  'variants',
+  'disordered_regions', // data coming from (?)
```

Comment:
```suggestion
  'intrinsically_disordered_regions',
```

It's coming for the protein endpoint with the `extra_features` modifier, e.g. `/api/protein/UniProt/P04637/?extra_features`.